src/components/ParserComponent.tsx

```diff
@@ -354,7 +367,7 @@ export default class ParserComponent extends React.Component<ParserInterface,
               this.sortData(this.state.data);
             } catch (e) {
               alert('data is EMPTY');
-              console.log('data is empty');
+              console.error('data is empty');
```

Comment:
should use e.toString() here because sortData & inferTypes throw different errors